integrations/nvidia/tests/test_document_embedder.py

```diff
@@ -4,7 +4,26 @@
 from haystack import Document
 from haystack.utils import Secret
 from haystack_integrations.components.embedders.nvidia import EmbeddingTruncateMode, NvidiaDocumentEmbedder
-from haystack_integrations.components.embedders.nvidia.backend import EmbedderBackend
+from haystack_integrations.components.embedders.nvidia.backend import EmbedderBackend, Model
+from requests_mock import Mocker
+
+
+@pytest.fixture
+def mock_local_models(requests_mock: Mocker) -> None:
```

Comment:
move this fixture to conftest.py to avoid duplicating it across files. see https://docs.pytest.org/en/stable/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files